api/libproject/generate_appengine_web_xml.sh

```diff
@@ -1,6 +1,29 @@
 #!/bin/bash
+
+for line in `awk '!/^ *#/ && NF' db/vars.env`; do
+  IFS='=' read -r var val <<< "$line"
+
+  if [ "$var" == "DB_HOST" ] && [ $OVERWRITE_WORKBENCH_DB_HOST ]; then
+    export DB_HOST=localhost
+    continue
+  fi
+
+  evaluatedString=$(echo $(eval "echo $val"))
+  echo "Exporting value $var=$evaluatedString"
+  export $var=$evaluatedString
+done
+
 set -euo pipefail
 IFS=$'\n\t'
 
-envsubst < src/main/webapp/WEB-INF/appengine-web.xml.template \
+cat src/main/webapp/WEB-INF/appengine-web.xml.template \
+  | sed "s|\${DB_DRIVER}|${DB_DRIVER}|" \
```

Comment:
This is a pretty verbose standin for `envsubst`. This is also not the only place we use envsubst in the codebase. Were you not able to include this dependency in the setup?

Reply:
this was added by @gjuggler as `envsubst` is not easy to install on Google macbooks?

Reply:
envsubst is a huge pita to install on OSX, especially in Google corp environment.

Agree this is verbose, but removes a dependency in the code-reload cycle. Also, the set of env variables substituted here hasn't changed for ~2y, so a bit of verbosity seems OK.

Other usages are outside of the main iteration loop, and are reasonable to be run from within the docker env where it's always available.

Reply:
OK, sounds fine to remove the dep. Please at least move the substitution up into the above loop instead so we don't need to hardcode all of these variable names though. I'd just copy the template and run `sed -i` on each turn of the loop. Or is `vars.env` not a superset of these?

Reply:
Ah, I hadn't thought of that, that would be nice. Though I don't think it's a huge deal to hard-code a few vars that rarely change. (It also makes this file show up more prominently when searching for these env variables, instead of the somewhat obscure dependency via vars.env.)

But probably best to do the cleaner solution w/ less maintenance burden overall. If Eric's willing to code it up :)